src/test/java/com/solvd/SearchTest.java

```diff
@@ -0,0 +1,44 @@
+package com.solvd;
+
+import com.solvd.pages.HomePage;
+import com.solvd.pages.ResultPage;
+import com.solvd.utils.ConfigReader;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.TimeUnit;
+
+public class SearchTest {
+    private final String driverPath = ConfigReader.getProperty("browser_path");
+    private WebDriver driver;
+    private HomePage homePage;
+    private ResultPage resultPage;
+
+    @BeforeTest
```

Comment:
@BeforeTest method is used before all test defined in testNG.xml between <test> tag. So inside test tag in xml you may have many classes. I think that you need @BeforeMethod here and @AfterMethod instead of @AfterTest.

Reply:
ok